src/transfer.ts

```diff
@@ -64,6 +70,116 @@ export async function ethTransactionToRawL2Transaction(
     },
   };
   const rawL2Tx = await buildProcess(abi, godwoker, ethTransaction, process);
+  if (typeof rawL2Tx === "string") {
+    throw new Error("rawL2Tx should not be string type.");
+  }
+  return rawL2Tx as RawL2Transaction;
+}
+
+/**
+ * for erc20 l2 transaction, the serialized data will be submit to web3 rpc poly_submitL2Transaction instead of gw_submitL2Transaction
```

Comment:
```suggestion
 * for erc20 l2 transaction, the serialized data will be submit to web3 rpc poly_submitL2Transaction instead of gw_submit_l2transaction
```